### DISCUSSION TITLE
How to generate coins address using type script

### DISCUSSION THREAD

**Discussion by HafeezRai** [trustwallet/wallet-core#1268] [Q&A] [answered]
is there any example code to generate coins addresses in typescript. Or can we get all method in typescript if that is true any example Please

**Comment by optout21** [comment, marked as answer]
We don't have TS sample code (only swift, kotlin, c++, go).  In fact, in our typescript we expose CoinType and models for signing.

https://github.com/trustwallet/wallet-core/tree/master/typescript/tests
